web/src/main/java/org/jboss/metadata/parser/jbossweb/Attribute.java

```diff
@@ -37,6 +37,8 @@
     // jboss-web attributes in alpha order
     CONFIG_PARSER_CLASS("configParserClass"),
 
+    FLUSH_ON_SESSION_INVALIDATION("flushOnSessionInvalidation"),
```

Comment:
I fully agree with you regarding name conventions, but this attribute is widely known and expected to work, so I can't see a way to rename it right now.